operator/src/main/java/oracle/kubernetes/operator/DomainType.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+public enum DomainType {
```

Comment:
Some how I got confused about this.   We have a ModelInImageDomainType and should we have a different name in here or change the MII to use the same class?

Reply:
Changed to DomainInPVType.